pkgs/tools/security/ssh-audit/default.nix

```diff
@@ -16,6 +16,12 @@ python3Packages.buildPythonApplication rec {
     sha256 = "sha256-g5h0A1BJqzOZaSVUxyi7IsCcrbto4+7+HpiVjFZy50Y=";
   };
 
+  outputs = [ "out" "man" ];
+  postInstall = ''
+    mkdir -p $man/share/man/man1
```

Comment:
Please use installManPage. That also created the directory automatically

Reply:
done